paperqa/agents/search.py

```diff
@@ -400,9 +401,8 @@ async def get_saved_object(
         return None
 
     def clean_query(self, query: str) -> str:
-        for replace in ("*", "[", "]", ":", "(", ")", "{", "}", "~", '"'):
-            query = query.replace(replace, "")
-        return query
+        # SEE:https://regex101.com/r/DoLMoa/3
```

Comment:
```suggestion
        # SEE: https://regex101.com/r/DoLMoa/3
```

Also, thanks for updating the values, it looks great